content/app/development/ux/texts/translation/_index.en.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Oversettelse av tekster
+description: Hvordan oversette tekster i applikasjonen.
+toc: true
+weight: 40
+tags: [translate-to-english]
+---
+
+Selve logikken rundt oversettelse av tekster benytter seg av tekst-ressursfiler. Se [Tekster](../texts) for mer informasjon om disse filene.
+Det er mulig å oversette tekster i applikasjonen ved å legge til flere tekst-ressursfiler. Altså én fil per språk.
+Under kan man se et eksempel på norsk og engelsk:
+
+`resource.nb.json`
+
+```json
+{
+  "language": "nb",
+  "resources": [
+    {
+      "id": "language.selector.label",
+      "value": "Språk"
+    },
+    {
+      "id": "language.full_name.nb",
+      "value": "Norsk bokmål"
+    },
+    {
+      "id": "language.full_name.en",
+      "value": "Engelsk"
+    }
+  ]
+}
+```
+
+`resource.en.json`
+
+```json
+{
+  "language": "en",
+  "languageDescription": "English",
+  "dropdownLabel": "Language",
```

Comment:
Are these options used or are they leftovers from the first design?

Reply:
Yes, removed the leftovers 👍